README.md

```diff
@@ -175,3 +176,11 @@ The models created by e-model-packager can be run easily using [EModelRunner](ht
 Useful links:
 [GitHub repo](https://github.com/BlueBrain/e-model-packager),
 [Documentation](https://e-model-packager.readthedocs.io/en/latest/).
+
+## emodel-generalisation
+
+Generalisation of neuronal electrical models on a morphological population with Markov Chain Monte-Carlo.
+
+Useful links:
+[GitHub repo](https://github.com/BlueBrain/emodel-generalisation),
+[Documentation](https://emodel-generalisation.readthedocs.io/en/latest/).
```

Comment:
I think you can also add the link and [citation](https://github.com/BlueBrain/emodel-generalisation?tab=readme-ov-file#emodel-generalisation) of the paper here.